android/modules/analytics/src/java/ti/modules/titanium/analytics/AnalyticsModule.java

```diff
@@ -208,7 +208,9 @@ public String getLastEvent()
 	// clang-format on
 	{
 		if (TiApplication.getInstance().isAnalyticsEnabled()) {
-			return analytics.getLastEvent().toString();
+			if (analytics.getLastEvent() != null) {
```

Comment:
Super nit-pick, but I'd prefer assignment to a temp variable here rather than call getLastEvent twice.